pkgs/development/python-modules/dm-tree/cmake.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/tree/CMakeLists.txt b/tree/CMakeLists.txt
+index 5e7eb43..7ad6d1a 100644
+--- a/tree/CMakeLists.txt
++++ b/tree/CMakeLists.txt
+@@ -50,62 +50,18 @@ if(APPLE)
+   set (CMAKE_FIND_FRAMEWORK LAST)
+ endif()
+ 
+-# Fetch pybind to be able to use pybind11_add_module symbol.
+-set(PYBIND_VER v2.6.2)
+-include(FetchContent)
+-FetchContent_Declare(
+-  pybind11
+-  GIT_REPOSITORY https://github.com/pybind/pybind11
+-  GIT_TAG        ${PYBIND_VER}
+-)
+-if(NOT pybind11_POPULATED)
+-    FetchContent_Populate(pybind11)
+-    add_subdirectory(${pybind11_SOURCE_DIR} ${pybind11_BINARY_DIR})
+-    include_directories(${pybind11_INCLUDE_DIR})
+-endif()
++find_package(pybind11 CONFIG REQUIRED)
+ 
+ # Needed to disable Abseil tests.
+ set (BUILD_TESTING OFF)
+ 
+ # Include abseil-cpp.
+-set(ABSEIL_VER 20210324.2)
+-include(ExternalProject)
+-ExternalProject_Add(abseil-cpp
+-  GIT_REPOSITORY    https://github.com/abseil/abseil-cpp.git
+-  GIT_TAG           ${ABSEIL_VER}
+-  PREFIX            ${CMAKE_SOURCE_DIR}/abseil-cpp
+-  CMAKE_ARGS       -DCMAKE_INSTALL_PREFIX=${CMAKE_SOURCE_DIR}/abseil-cpp
+-                   -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
+-                   -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+-                   -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+-                   -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+-                   -DCMAKE_POSITION_INDEPENDENT_CODE=${CMAKE_POSITION_INDEPENDENT_CODE}
+-                   -DLIBRARY_OUTPUT_PATH=${CMAKE_SOURCE_DIR}/abseil-cpp/lib
+-)
+-ExternalProject_Get_Property(abseil-cpp install_dir)
+-set(abseil_install_dir ${install_dir})
+-include_directories (${abseil_install_dir}/include)
+-
++find_package(absl REQUIRED)
+ 
```

Comment:
this doesn't have to be a blocker for this PR, but it might be nice to upstream the package with a `find_package ... fallback to external` type of workflow.

Reply:
Ack, will see if I can upstream it separately, will keep the patch meanwhile.